doc/contributing/docs/style.rst

```diff
@@ -174,13 +174,38 @@ Spelling and punctuation
 US vs British spelling
 ~~~~~~~~~~~~~~~~~~~~~~
 
-We use the US English spelling.
+Use the US English spelling.
 
 Check your spelling and punctuation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Consider checking spelling, grammar, and punctuation with special tools.
 
+Dashes
+~~~~~~
+
+There is no need to insert special symbols like dashes, quotation marks, or apostrophes.
```

Comment:
It's less about need and more about consistency in the sources and in the translation memory.

Reply:
And we can say the same about quotes